### DISCUSSION TITLE
Printing a page, or printing to PDF

### DISCUSSION THREAD

**Discussion by stellarfinn** [Budibase/budibase#7096] [Help]
When I try to print a page the layout is all out of whack. I'm not sure if it's hidden padding, margins or what. Any thoughts? See attached image. 

I'd like to be able to generate spec sheets from our database but they need to be downloadable or printable.

PAGE VIEW IN BROWSER
![Page as viewed in browser](https://user-images.githubusercontent.com/99847728/182909967-be7fdba4-6663-461a-956b-034e4e299b71.jpg)

PAGE VIEW IN PRINTER WINDOW
![Page as viewed in printer window](https://user-images.githubusercontent.com/99847728/182909986-29ab5c9a-9e34-4d62-a2dc-228b342140e5.jpg)

**Comment by mervyn15** [comment]
Use print friendly pdf chrome extension

**Reply by stellarfinn** [reply]
A third party solution is great but I'm also interested in understanding why this is happening.

**Reply by melohagan** [reply]
@stellarfinn Have you tried playing around with the width/positioning of the parent screen container?

**Reply by mahxds** [reply]
@melohagan  i tried that and i have to input 30% bigger values... dunno why
